packages/calcite-components/src/components/stepper-item/stepper-item.scss

```diff
@@ -73,6 +73,10 @@
 // .stepper-item-header / content
 :host .stepper-item-header {
   @apply flex cursor-pointer items-start;
+
+  &:active {
+    @apply bg-foreground-3;
```

Comment:
We haven't done stepper item yet but since you're here, I think this should be `background-color: var(--calcite-stepper-item-header-background-color-active, var(--calcite-color-foreground-3);`

Then add the jsdocs

Reply:
@alisonailea Kitty and me discussed this, the tokens work is out of scope for this PR. We'll merge this as is and take care of the token in its own issue.

cc @geospatialem